libbpf-rs/src/perf_buffer.rs

```diff
@@ -220,10 +220,30 @@ impl<'b> PerfBuffer<'b> {
     }
 }
 
-impl<'b> Drop for PerfBuffer<'b> {
+// SAFETY: `perf_buffer` objects can safely be polled from any thread.
+unsafe impl Send for PerfBuffer<'_> {}
+
+impl Drop for PerfBuffer<'_> {
     fn drop(&mut self) {
         unsafe {
             libbpf_sys::perf_buffer__free(self.ptr);
         }
     }
 }
+
+#[cfg(test)]
```

Comment:
What are your thoughts on moving the tests to the libbpf-rs/tests directory?

Reply:
I don't feel strongly about it, but in general I'd think it makes the most sense to have integration style tests (that test the public API surface) reside inside `libbpf-rs/tests`. Also see [here](https://doc.rust-lang.org/book/ch11-03-test-organization.html). This one is arguably only a build time check anyway, so it feels most natural to keep it closest to the source code, at least to me. Happy to move it if that's what you prefer, though.